Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use alpine for a smaller image size and install only the required packages
-FROM python:3.8-slim-buster
+FROM python:3.11.0-slim
```

Comment:
Shouldn't this be `3.11.1` to match the `ci.yml` file above?
Also are we OK to be using `bullseye` which is the default if you don't specify?  I had a problem with a dependency when switching to `bullseye` for one of the other repos.  Perhaps not an issue here.

Reply:
Yes, I'll modify it to match the specific version, and as for the moment has been using bullseye I'll use 3.11.1-slim-bullseye.